src/libraries/System.Net.Http/src/System/Net/Http/HttpMethod.cs

```diff
@@ -19,8 +22,8 @@ public class HttpMethod : IEquatable<HttpMethod>
         private static readonly HttpMethod s_deleteMethod = new HttpMethod("DELETE", http3StaticTableIndex: H3StaticTable.MethodDelete);
         private static readonly HttpMethod s_headMethod = new HttpMethod("HEAD", http3StaticTableIndex: H3StaticTable.MethodHead);
         private static readonly HttpMethod s_optionsMethod = new HttpMethod("OPTIONS", http3StaticTableIndex: H3StaticTable.MethodOptions);
-        private static readonly HttpMethod s_traceMethod = new HttpMethod("TRACE");
-        private static readonly HttpMethod s_patchMethod = new HttpMethod("PATCH");
+        private static readonly HttpMethod s_traceMethod = new HttpMethod("TRACE", -1);
+        private static readonly HttpMethod s_patchMethod = new HttpMethod("PATCH", -1);
```

Comment:
I think you meant to make `_http3Index` non-nullable, and to set it to `-1` in the default ctor.

Otherwise, initializing these to `-1` appears to have no meaningful effect.

Reply:
Initializing to `-1` means that it's a singleton instance with no HTTP3 static
Initializing to `null` means it's not a singleton instance

we could use `-2` instead of null if that's what you are suggesting

Reply:
Ahhh I see the distinction now. Thanks. Okay, Is good to merge.